test/system/410-selinux.bats

```diff
@@ -50,6 +50,14 @@ function check_label() {
     check_label "--systemd=always" "container_init_t"
 }
 
+@test "podman selinux: init container with --security-opt type" {
+    check_label "--systemd=always --security-opt=label=type:spc_t" "spc_t"
```

Comment:
This test fails (root and rootless) on my laptop: gets `container_init_t` instead of `spc_t`.